src/cli/cli_br.cpp

```diff
@@ -435,6 +435,66 @@ template <> otError Br::Process<Cmd("prefixtable")>(Arg aArgs[])
     return error;
 }
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+template <> otError Br::Process<Cmd("pd")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    /**
+     * @cli br pd (enable,disable)
+     * @code
+     * br pd enable
+     * Done
+     * @endcode
+     * @code
+     * br pd disable
+     * Done
+     * @endcode
+     * @cparam br pd @ca{enable|disable}
+     * @par api_copy
+     * #otBorderRoutingDhcp6PdSetEnabled
+     *
+     */
+    if (Interpreter::GetInterpreter().ProcessEnableDisable(aArgs, otBorderRoutingDhcp6PdSetEnabled) == OT_ERROR_NONE)
+    {
```

Comment:
We already have `otBorderRoutingDhcp6PdSetEnabled`, so we can call it now?

Reply:
I did not see a call of `otBorderRoutingDhcp6PdSetEnabled(GetInstancePtr(), otBorderRoutingDhcp6PdSetEnabled)` here?

Am I missing anything?

Reply:
Ah, I misunderstood the function, sorry.